backend/services/background_job_handlers.py

```
@@ -0,0 +1,92 @@
+        "telephone": reservation.telephone,
+    }
+
+
+def handle_portale_alloggi_submit(db_session, payload):
+    """Process Portale Alloggi real submission job."""
+    reservation_id = payload.get("reservation_id")
+    user_id = payload.get("user_id")
+
+    reservation = db_session.query(Reservation).filter(Reservation.id == reservation_id).first()
+    if not reservation:
+        raise JobExecutionError("Reservation not found")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)